docs/install_devel_build.md

```diff
@@ -0,0 +1,43 @@
+# Install Development Build
+
+If you are interested in helping the development of this tool, or you would like
+to get the cutting-edge version of this tool, you can install this tool via
+conda.
+
+To do this, ensure you have Miniconda/Anaconda installed on your system. You can
+download miniconda
+on [their official website](https://docs.anaconda.com/miniconda/).
+
+
+1. Clone this repository from GitHub:
+   ```bash
+   git clone git@github.com:UBC-MDS/fixml.git
+   ```
+
+2. Create a conda environment:
+
+    ```bash
+    conda env create -f environment.yml
+    ```
+
+3. Activate the newly created conda environment (default name `fixml`):
+
+    ```bash
+    conda activate fixml
+    ```
+
+4. Use `poetry` which is preinstalled in the conda environment to create a local package install:
+
+    ```bash
+    poetry install
+    ```
+
```

Comment:
```suggestion
export "OPENAI_API_KEY={your-openai-api-key}" > .env
```

Reply:
There are more than one way to set up the key. Also, the key needs to be upgraded from free tier. 

Since this guide is targeting developers which I assume would have basic reading comprehension skills, I would suggest to link back to the api key section in installation.md instead of appending a simple export command without explanations.

Reply:
Changed in 8dd818a.